store/tx_manager.go

```diff
@@ -140,9 +140,14 @@ func (txm *EthTxManager) CreateTxWithGas(to common.Address, data []byte, gasPric
 }
 
 // CreateTxWithEth signs and sends a transaction with some ETH to transfer.
-func (txm *EthTxManager) CreateTxWithEth(to common.Address, value *assets.Eth) (*models.Tx, error) {
-	ma, err := txm.nextAccount()
-	if err != nil {
+func (txm *EthTxManager) CreateTxWithEth(to common.Address, value *assets.Eth, from ...common.Address) (*models.Tx, error) {
+	ma := &ManagedAccount{}
+	var err error
+	if len(from) == 1 {
+		if ma = txm.getAccount(from[0]); ma == nil {
```

Comment:
I've gone back and forth on whether or not we want a variadic parameter to support an optional `from` parameter. In the spirit of compile time correctness over run time errors, let's move away from the variadic parameter by doing the following:

1. In https://github.com/smartcontractkit/chainlink/blob/master/web/transfer_controller.go#L26, if from is empty, fill with `TxManager#NextActiveAccount()`
2. Change `CreateTxWithEth` to CreateTxWithEth(**from**, to common.Address, value *assets.Eth) (*models.Tx, error)

Let me know if you have any thoughts or objections.

Reply:
We'll want `CreateTxWithEth` to return an error if the `FromAddress` doesn't exist or is not an available address in the tx manager.

Reply:
Okay i'll finalize this pr soon thanks